dir.props

```diff
@@ -155,6 +155,7 @@
     <!-- Central place to set the versions of all nuget packages produced in the repo -->
     <PackageVersion Condition="'$(PackageVersion)' == ''">1.2.0</PackageVersion>
     <WindowsAPISetPackageVersion>1.0.1</WindowsAPISetPackageVersion>
+    <SystemNativePackageVersion>4.4.0-beta-24721-02</SystemNativePackageVersion>
     
```

Comment:
It looks like I have to use `CoreFxExpectedPrerelease` and `ExternalExpectedPrerelease`. Any advice here would be helpful.